_topics/en/pcac.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Payments Contingent on Anything Computable (PCAC)
```

Comment:
Will probably rename this to Accountable Computing Contracts (ACC); see https://twitter.com/hrdng/status/1742597010967118177